test/common/testlib.py

```diff
@@ -230,15 +230,21 @@ def blur(self, selector):
 
     def key_press(self, keys):
         for k in keys:
-            if k.isalnum():
+            if k == "Backspace":
+                self.cdp.invoke("Input.dispatchKeyEvent", type="keyDown", windowsVirtualKeyCode=8)
```

Comment:
urgh magic numbers.. would `code="Delete"` work? (See https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)

Reply:
I tried `code="Backspace"` but that didn't do anything.  I can try "Delete".  8 isn't that magic, it's C-h or "bs" in ASCII, and everyone should know that when they come out of hacker pre-school. :-)

What about `ord("\b")`?

Reply:
Ah, that's just the ASCII code, right. Yeah, for some reason `Backspace` isn't defined there, just `Delete`. No need to sink more time on this, though, it already took enough :)